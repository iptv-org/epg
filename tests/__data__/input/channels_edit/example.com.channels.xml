<?xml version="1.0" encoding="UTF-8"?>
<channels>
  <channel site="example.com" lang="" xmltv_id="" site_id="140">CNN International Europe</channel>
  <channel site="example.com" lang="" xmltv_id="" site_id="141">CNN International</channel>
  <channel site="example.com" lang="" xmltv_id="" site_id="142">Bravo</channel>
</channels>
