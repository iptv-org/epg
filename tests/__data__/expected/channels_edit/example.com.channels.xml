<?xml version="1.0" encoding="UTF-8"?>
<channels>
  <channel site="example.com" lang="" xmltv_id="CNNInternational.us@Europe" site_id="140">CNN International</channel>
</channels>
